homeassistant/components/sws12500/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    }
+  }
+}
```

Comment:
This is just some random `strings.json`. It should have all the keys and strings used for your integration config flow and options flow. It's similar to the `en.json` translation, but can use references to the standard HA strings, so the translation process is not repeated for the common sentences